pkg/skaffold/schema/util/util.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+type VersionedConfig struct {
```

Comment:
why do we need this struct? version is already part of the config. 
can you make it an interface and then implement 

GetVersion() string for each config object?

Reply:
yeah that works, updated